### PR TITLE
add copyright

### PR DESCRIPTION
**Pull request by chenzhihao**
`make` passes now

### CODE REVIEW

File: instrumentation/github.com/aws/aws-sdk-go/service/otels3/s3_test.go [hunk 1]
```diff
@@ -170,3 +189,28 @@ func Test_instrumentedS3_PutObjectWithContext(t *testing.T) {
 		})
 	}
 }
+
+func TestNewInstrumentedS3Client(t *testing.T) {
```
Comment (Tenaria) [inline instrumentation/github.com/aws/aws-sdk-go/service/otels3/s3_test.go]: were you meant to commit this? 😛

File: .github/dependabot.yml [hunk 2]
```diff
@@ -307,6 +307,26 @@ updates:
     schedule:
       interval: "weekly"
       day: "sunday"
+  -
+    package-ecosystem: "gomod"
```
Comment (Tenaria) [inline .github/dependabot.yml:311]: just for me: what does this actually do?

File: instrumentation/github.com/aws/aws-sdk-go/service/otels3/s3_test.go (hunk 1 above)
Comment (chenzhihao) [reply instrumentation/github.com/aws/aws-sdk-go/service/otels3/s3_test.go]: 🤦

File: .github/dependabot.yml (hunk 2 above)
Comment (chenzhihao) [reply .github/dependabot.yml:311]: I think the Otel repo is using https://dependabot.com/, and this is for the configuration stuff.

File: go.mod [hunk 3]
```diff
@@ -1,4 +1,4 @@
-module go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-sdk-go/service
+module go.opentelemetry.io/contrib
```
Comment (chenzhihao) [inline go.mod:1]: sorry I messed it up before.

**Review by Tenaria — APPROVED** [review]

**Pull request merged**